test/test_sbol2_sbol3_direct.py

```diff
@@ -17,7 +20,7 @@ class TestDirectSBOL2SBOL3Conversion(unittest.TestCase):
     # TODO: turn on validation
 
     # J23101.xml is not SBOL compliant. Leaving conversions involving it for after the compliant converter is done
-    '''
+    
```

Comment:
Don't add extra whitespace against python style; use pycodestyle to check your code.